audio/FAudio/Portfile

```diff
@@ -46,6 +46,14 @@ variant ffmpeg requires wma description {Legacy compatibility variant} {}
 #    file copy ${workpath}/build/facttool ${destroot}${prefix}/bin/
 #}
 
+# FAudio doesn't generate a pkgconfig configuration file, so add our own
```

Comment:
Can you add a comment here with a link to the upstream bug report or pull request, please?

Reply:
Sure I can add a link tonight

The comment mentioning a lack of pc file can be found in this link here
https://github.com/FNA-XNA/FAudio/issues/159#issuecomment-531532251

Reply:
So you've added that link as a comment now, but that link isn't a bug report or pull request about this problem. It's a comment in a different closed ticket. You should file a new bug report or pull request with them and add that link here.

Reply:
Your correct I should have re-read your comment. Sorry about that

I’ve opened an issue related to this now https://github.com/FNA-XNA/FAudio/issues/224

Once my system is free I’ll force push the change